js/components/portal-dashboard/header-menu-item.tsx

```diff
@@ -20,8 +20,17 @@ export class HeaderMenuItem extends React.PureComponent<IProps> {
     const { menuItem, colorTheme } = this.props;
     const colorClass = colorTheme ? css[colorTheme] : "";
     return (
-      <div className={`${css.menuItem} ${colorClass}`} onClick={this.handleSelect} data-cy={menuItem.dataCy}>
-        <CheckIcon className={`${css.check} ${colorClass} ${menuItem.selected ? css.selected : ""}`} />
+      <div
+        className={`${css.menuItem} ${colorClass}`}
+        onClick={this.handleSelect}
+        data-cy={menuItem.dataCy}
+        data-testid={menuItem.dataCy}
+      >
+        <CheckIcon
+          className={`${css.check} ${colorClass} ${menuItem.selected ? css.selected : ""}`}
+          data-cy={`${menuItem.dataCy}-selected`}
+          data-testid={`${menuItem.dataCy}-selected`}
+        />
```

Comment:
Do we need both `data-cy` and `data-testid` attributes? I would've thought we just need one, but maybe there's a purpose for adding both?